sci-misc/boinc/files/boinc.init

```diff
@@ -11,10 +11,10 @@ depend() {
 
 
 create_work_directory() {
-	if [[ ! -d "${RUNTIMEDIR}" ]]; then
+	if [ ! -d "${RUNTIMEDIR}" ]; then
```

Comment:
By the way, the whole function looks like a case for `checkpath`, or tmpfiles.d (the latter preferred as more portable).